Pico/serial.h

```diff
@@ -0,0 +1,15 @@
+#ifndef serial_h
+#define serial_h
+
+#include <Arduino.h>
+
+class SerialCommunication {
+    public:
+        SerialCommunication(HardwareSerial& serial);
+        float Receive(uint8_t signal);
+
+    private:
+        HardwareSerial& serial_;
```

Comment:
Did this class compiled without warnings? It looks great and minimalistic, just wanted to be sure it works

Reply:
Working now and tested